xml/adm_sudo.xml

```diff
@@ -0,0 +1,611 @@
+      <screen>
+&prompt.user;ENVVAR=test env | grep ENVVAR
+ENVVAR=test
+&prompt.user;ENVVAR=test sudo env | grep ENVVAR
+root's password:
+<co xml:id="co.sudo.env.novar"/>
+&prompt.user;
+      </screen>
+      <calloutlist>
+        <callout arearefs="co.sudo.env.novar">
+          <para>
+            The empty output shows that the environment variable <envar>ENVVAR</envar>
+            did not exist in the context of the command run with &sudo;.
+          </para>
+        </callout>
+      </calloutlist>
+
+      <para>This behaviour can be changed by the <literal>env_reset</literal> option, see <xref linkend="tab.adm.sudo.options"/>.</para>
+    </sect2>
+  </sect1>
+
+  <sect1 xml:id="sec.sudo.conf">
+    <title>Configuring &sudo;</title>
+
+    <para>
+      &sudo; is a very flexible tool with extensive configuration.
+    </para>
+
+    <note>
+      <title>Locked out of sudo</title>
+      <para>
+        Should you lock yourself out of &sudo; by mistake, you can use <command>su -</command>
+        and the &rootuser; password to get a root shell.
+      </para>
+    </note>
+
+    <sect2 xml:id="sec.sudo.conf.edit">
+      <title>Editing the configuration files</title>
+      <para>
+        The main policy configuration file for &sudo; is <filename>/etc/sudoers</filename>.
+        As it is possible to lock yourself out of the system if there are any errors in that file, it is
+        strongly recommended to use <command>visudo</command> for editing.
+        It will prevent simultaneous changes to the opened file and check for syntax errors before saving.
+      </para>
+      <para>
+        Despite its name, you can also use editors other than vi by setting the <envar>EDITOR</envar>
+        environment variable, for example:
+      </para>
+      <screen>sudo EDITOR=<replaceable>/usr/bin/nano</replaceable> visudo</screen>
+
+      <para>
+        However, the <filename>/etc/sudoers</filename> file itself is supplied by the system packages and
+        modifications may break on updates. Therefore, it is recommended to put any custom configuration into
+        the <filename>/etc/sudoers.d/</filename> directory. Any file in there is automatically included.
+        To edit a file in that subdirectory, run:
+        </para>
+      <screen>sudo visudo -f /etc/sudoers.d/<replaceable>NAME</replaceable></screen>
+
+      <para>Or with <command>nano</command> as editor:</para>
+      <screen>sudo EDITOR=<replaceable>/usr/bin/nano</replaceable> visudo -f /etc/sudoers.d/<replaceable>NAME</replaceable></screen>
+
+      <note>
+        <title>Ignored files in <filename>/etc/sudoers.d</filename></title>
+        <para>
+          The <literal>#includedir</literal> command in <filename>/etc/sudoers</filename>,
+          used for <filename>/etc/sudoers.d</filename>, ignores files that end in <literal>~</literal>
+          or <emphasis>contain</emphasis> a <literal>.</literal>.
+        </para>
```

Comment:
Files which names end in &lt;literal>~ &lt;/literal> (tilde) or conatin a &lt;literal>. &lt;/literal> (period) are ignored by default.